code/modules/mob/living/carbon/xenomorph/castes/runner/abilities_runner.dm

```diff
@@ -143,6 +143,8 @@
 /datum/action/xeno_action/activable/pounce/proc/prepare_to_pounce()
 	if(owner.layer == XENO_HIDING_LAYER) //Xeno is currently hiding, unhide him
 		owner.layer = MOB_LAYER
+		for(var/datum/action/xeno_action/xenohide)
```

Comment:
Uh what the fuck? How is this a valid for loop? This loops thru the whole WORLD STOP IT

Reply:
no warning by the way

Reply:
oh wait i'm dumb, why it even compiles